lib/warnings.js

```diff
@@ -57,3 +63,4 @@ function emitWarning (code) {
 }
 
 module.exports = { codes, createWarning, emitWarning }
+module.exports[Symbol.for('internals')] = { emittedWarnings }
```

Comment:
Is this just for tests?

Reply:
Yes it is. I need to remove one of the warnings in tests.

Reply:
Then at the very least, namespace it like `Symbol.for('fastify.warnings-internals')`.

Reply:
oh, a global map..

Reply:
Let's try not to use global maps for stuff that's not needed. Please use a private `Symbol`.

Reply:
Oh man, you're right. I always get confused with the two syntaxes. I'll change that. I think there is also a similar (existing) one in Content Type. I'll fix that as well.